lib/Db/PublicationType.php

```diff
@@ -122,4 +123,33 @@ public function jsonSerialize(): array
 
 		return $array;
 	}
+
+	/**
+	 * Generate a JSON-Schema definition for the data field of a publication.
+	 *
+	 * @param IURLGenerator $urlGenerator An URL generator to generate the identifier of the schema.
+	 *
+	 * @return object The JSON-Schema object defining the data field of a publication.
+	 */
+	public function getSchema(IURLGenerator $urlGenerator): object
```

Comment:
1. https://github.com/ConductionNL/openregister/blob/2fded415ce4b87bb9d5733a10db7d9572259a2c9/lib/Db/Schema.php#L80

2. https://github.com/ConductionNL/openregister/blob/2fded415ce4b87bb9d5733a10db7d9572259a2c9/lib/Db/Schema.php#L148

Hierbij even de vraag in hoe ver we willen dat OpenRegister Schema en OpenCatalogi PublicationType het zelfde werken @rubenvdlinde

Reply:
Dit werkt uiteraard 2 kanten op, wat vanuit OpenCatalogi willen we ook zo hebben in OpenRegister
bijv. de switch die we hebben bij OpenCatalogi PublicationType jsonserialize is 'verder' / heeft meer code dan die in OpenRegister Schema